src/System.Numerics.Tensors/System/Numerics/ArrayTensorExtensions.cs

```diff
@@ -6,144 +6,141 @@ namespace System.Numerics
     public static class ArrayTensorExtensions
     {
         /// <summary>
-        /// Creates a copy of this single-dimensional array as a DenseTensor<T>
+        /// Creates a copy of this single-dimensional array as a DenseTensor&lt;T&gt;
         /// </summary>
-        /// <typeparam name="T"></typeparam>
-        /// <param name="array"></param>
-        /// <param name="reverseStride"></param>
-        /// <returns></returns>
-        public static DenseTensor<T> ToTensor<T>(this T[] array, bool reverseStride = false)
+        /// <typeparam name="T">Type contained in the array to copy to the DenseTensor&lt;T&gt;.</typeparam>
+        /// <param name="array">The array to create a DenseTensor&lt;T&gt; from.</param>
+        /// <returns>A 1-dimensional DenseTensor&lt;T&gt; with the same length and content as <paramref name="array"/>.</returns>
+        public static DenseTensor<T> ToTensor<T>(this T[] array)
         {
-            return new DenseTensor<T>(array, reverseStride);
+            return new DenseTensor<T>(array);
         }
 
         /// <summary>
-        /// Creates a copy of this two-dimensional array as a DenseTensor<T>
+        /// Creates a copy of this two-dimensional array as a DenseTensorr&lt;T&gt;
         /// </summary>
-        /// <typeparam name="T"></typeparam>
-        /// <param name="array"></param>
-        /// <param name="reverseStride"></param>
-        /// <returns></returns>
+        /// <typeparam name="T">Type contained in the array to copy to the DenseTensor&lt;T&gt;.</typeparam>
+        /// <param name="array">The array to create a DenseTensor&lt;T&gt; from.</param>
+        /// <param name="reverseStride">False (default) to indicate that the first dimension is most minor (closest) and the last dimension is most major (furthest): row-major.  True to indicate that the last dimension is most minor (closest together) and the first dimension is most major (furthest apart): column-major.</param>
```

Comment:
A thing I found confusing about this description is that it says the default is "first dimension is most minor" and is "row major".  Given that the rows are the first dimension - "row major" would mean that the first dimension is the "most major", right?

Reply:
Typo when I was rewording this a few times.  Will fix.